axelor-account/src/main/java/com/axelor/apps/account/service/batch/BatchReimbursementImport.java

```diff
@@ -97,12 +97,18 @@ protected void process() {
       Map<List<String[]>, String> data = null;
 
       try {
-        rejectImportService.createFilePath(accountConfig.getReimbursementImportFolderPathCFONB());
+
+        String dataImportDir = appBaseService.getFileUploadDir();
+
+        String reimbursementImportFolderPathCFONB =
+            dataImportDir + accountConfig.getReimbursementImportFolderPathCFONB();
+
+        rejectImportService.createFilePath(reimbursementImportFolderPathCFONB);
 
         data =
             rejectImportService.getCFONBFileByLot(
-                accountConfig.getReimbursementImportFolderPathCFONB(),
-                accountConfig.getTempReimbImportFolderPathCFONB(),
+                reimbursementImportFolderPathCFONB,
+                dataImportDir + accountConfig.getTempReimbImportFolderPathCFONB(),
```

Comment:
Pass null value in parameter if accountConfig.getTempReimbImportFolderPathCFONB() is null